src/api/app/models/package.rb

```diff
@@ -1320,7 +1320,10 @@ def abort_build(params)
 
   def backend_build_command(command, params)
     begin
-      Suse::Backend.post("/build/#{URI.escape(project.name)}?cmd=#{command}&#{params.to_query}", '')
+      # Note: This list needs to keep in sync with the backend code
+      permitted_params = params.permit(:repository, :arch, :package, :code, :wipe)
```

Comment:
@ChrisBr This would mean to update [this](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/package.rb#L1309) and few other places  as well. So it would take some bigger effort. I would prefer to do it independently from this PR.